### ISSUE TITLE
vkGetSwapchainImagesKHR incorrect argument descriptions

### ISSUE THREAD

**Issue by philiptaylor** [KhronosGroup/Vulkan-Docs#292]
https://www.khronos.org/registry/vulkan/specs/1.0-wsi_extensions/xhtml/vkspec.html#vkGetSwapchainImagesKHR says:

>```
VkResult vkGetSwapchainImagesKHR(
    VkDevice                                    device,
    VkSwapchainKHR                              swapchain,
    uint32_t*                                   pSwapchainImageCount,
    VkImage*                                    pSwapchainImages);
```
> ...
> * `pSwapchainImageCount` is a pointer to an integer related to the number of format pairs available or queried, as described below.
> * `pSwapchainImages` is either `NULL` or a pointer to an array of `VkSwapchainImageKHR` structures. 
 
The description of `pSwapchainImageCount` is wrong, it should say "swapchain images" not "format pairs".

The description of `pSwapchainImages` is wrong since `VkSwapchainImageKHR` doesn't exist, it should say "array of `VkImage` handles".

**Comment by oddhack** [comment]
This was partly fixed in the 1.0-VK_KHR_swapchain branch but the second part hasn't been fixed yet, nor has it been merged to 1.0-wsi_extensions, so there's more to do here.

**Comment by ianelliottus** [comment]
Sorry I missed the 2nd part.  I just created a commit for that.

**Comment by oddhack** [comment]
Should be fixed in the 1.0.24 update.
